docs/tasks.md

```diff
@@ -91,13 +101,16 @@ Add discovered PDRs to a queue
 ---
 
 ### [@cumulus/sf-sns-report](https://github.com/nasa/cumulus/tree/master/tasks/sf-sns-report)
-Publish ingest notifications to SNS
 
+Broadcasts an incoming Cumulus message to SNS
```

Comment:
Oh, or did you re-generate this doc using the script? Because I see this description is in the `package.json` for this task

Reply:
Probably a combination of both.  I was on a roll updating lint failures and missed this was autogenerated.

Reply:
In either case, this changeset is going to be screwy until Menno merges the PR I mentioned in the description.  If you'd like to review before then, we'd need to re-target it to that PR as I based this one off of it to avoid merge madness.

Reply:
Nah, I just got ahead of myself